src/gmail.d.ts

```diff
@@ -708,7 +708,19 @@ interface GmailTools {
     i18n(label: string): string;
     add_toolbar_button(content_html: string, onClickFunction: Function, styleClass: string): JQuery;
     add_right_toolbar_button(content_html: string, onClickFunction: Function, styleClass: string): JQuery;
-    add_compose_button(composeWindow: GmailDomCompose, content_html: string, onClickFunction: Function, styleClass: string): JQuery;
+    add_compose_button(composeWindow: GmailDomCompose, content_html: string, onClickFunction: Function, styleClass?: string): JQuery;
+    add_more_send_option(
+      composeWindow: GmailDomCompose,
+      content_html: string,
+      onClickFunction: JQuery.TypeEventHandler<
+        HTMLDivElement,
+        null,
+        HTMLDivElement,
+        HTMLDivElement,
+        'click'
+      >,
```

Comment:
I think this type is more complicated than it needs to be.

What we pass in as an on-click function should just be an anonymous `() => void` or `Function`.

Reply:
Yes, I can simplify though the type is actually `(event: JQuery.Event) => void`

Reply:
I'd prefer it to be either be unwrapped to the DOM-native, or just a simple `() => void`, just like the rest of the `onClickHandlers` we have all around :)

Reply:
Unwrapping would be a semantic change rather than just a tweaking of the declared types. i.e. for `add_compose_button`, the `onClickFunction` is passed directly to jquery's `click`: https://github.com/KartikTalwar/gmail.js/blob/b7e5b814e48de270fd33a04f85b4682bb7676892/src/gmail.js#L3590

unwrapping to DOM-native would be a breaking change to the code it self. I think for consistency with other methods, the `onClickFunction` here should match. Further the return type for all of the `add_XXX_button` is `JQuery` rather than the `HTMLElement`. It seems reasonable that we are taking in and return JQuery wrapper rather than native DOM. 

Similarly, for other methods, `onClickFunction` is declared as `Function`:
https://github.com/KartikTalwar/gmail.js/blob/b7e5b814e48de270fd33a04f85b4682bb7676892/src/gmail.d.ts#L709-L711
`(event: JQuery.Event) => void` is just a more precise type than currently specified.

Reply:
Ok, I can change it to `Function`. That being said, I think if I were to make the function `(event: JQuery.Event) => void`, users can still pass in a `() => void` i.e. that would not be a breaking change as the new type is more permissive.